ccmlib/cluster.py

```diff
@@ -646,14 +682,14 @@ def __update_topology_files(self):
             self.__update_topology_using_rackdc_properties()
 
     def __update_topology_using_toplogy_properties(self):
-        dcs = [('default', 'dc1')]
+        dcs = [('default', 'dc1', 'r1')]
```

Comment:
Why r1 and not RAC1 ?

It contradicts the default in other places of the code.

Also we should be careful not to change this default, it affects multiple users of CCM, not just dtest.

Reply:
See my comment below

Reply:
anayway it is used for `PropertyFileSnitch` snitch (don't mix with `GossipingPropertyFileSnitch`) - so it is never used anyway.